src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public void getRandomBall(int num) {
```

Comment:
```suggestion
    public Ball getRandomBall(int num) {
```
this method should return a ball